components/ChatPage/ChatExcelTab.tsx

```diff
@@ -0,0 +1,36 @@
+import { useDialogueContext } from '@/app/context/dialogue';
+import ExcelUpload from './ExcelUpload';
+import { useSearchParams } from 'next/navigation';
+import { LinkOutlined } from '@ant-design/icons';
+
+interface Props {
+  onComplete?: () => void;
+}
+
+function ChatExcelTab({ onComplete }: Props) {
+  const searchParams = useSearchParams();
+  const id = searchParams.get('id');
+  const scene = searchParams.get('scene');
+  const { currentDialogue } = useDialogueContext();
+
+  if (scene !== 'chat_excel') return null;
+
+  return (
+    <div className="w-full py-4 flex items-center border-b border-gray-100">
+      <div className="w-full lg:w-4/5 xl:w-3/4 mx-auto">
+        {currentDialogue ? (
+          <div className="flex overflow-hidden rounded">
+            <div className="flex items-center justify-center px-3 py-2 bg-gray-600">
+              <LinkOutlined className="text-white" />
+            </div>
+            <div className="bg-gray-100 px-3 py-2 text-xs rounded-tr rounded-br dark:text-gray-800">{currentDialogue.select_param}</div>
+          </div>
+        ) : (
+          <ExcelUpload convUid={id as string} chatMode={scene} onComplete={onComplete} />
```

Comment:
`chatMode` --> `scene`, the name `chatMode`  is very inappropriate and will be changed later.